### DISCUSSION TITLE
Recover scheduler

### DISCUSSION THREAD

**Discussion by danies8** [quartznet/quartznet#2316] [Q&A]
Hi,
When job scheduler stop working after running few days  on worker service.
How to recover ?  
1.Stop/Start service - not help.
2.Restart the machine of the service ?
Thanks,

**Comment by danies8** [comment]
Does the jobs keep in memory?

**Comment by danies8** [comment]
@lahma 
any answer please?

**Reply by lahma** [reply]
Sorry, I don't have the answer.

**Reply by danies8** [reply]
Does jobs keep in memory?

**Reply by danies8** [reply]
I do not used DB.

**Reply by danies8** [reply]
Does jobs keep in memory or cashed in computer?

**Reply by danies8** [reply]
@lahma
Can you please answer ?

**Reply by danies8** [reply]
@lahma
Why when i stoped service and started from cratch the scheduler not fire the jobs ?

**Comment by jacekkulis** [comment]
Short answer is no. Read more in the [documentation](https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/job-stores.html#ado-net-job-store-adojobstore).
